pyscf/md/integrators.py

```diff
@@ -322,6 +358,10 @@ def check_sanity(self):
         assert self.veloc is not None
         assert self.veloc.shape == (self.mol.natm, 3)
         assert self.scanner is not None
+        
```

Comment:
I recommend overloading the `check_sanity` func in the `NVTBerendson` and then calling the parent function within that. Something like:

```python
def check_sanity(self):
    assert self.T > 0
    assert self.taut is not None
    self.super().check_sanity()
```

Reply:
Done!